Assets/Modules/UI/Scripts/OldMainMenu.cs

```diff
@@ -8,7 +8,7 @@
 /// <summary>
 /// TODO
 /// </summary>
-public class MainMenu : MonoBehaviour
+public class OldMainMenu : MonoBehaviour
```

Comment:
Delete the file

Reply:
Toujours présent dans l'ancienne scène, donc je l'aurai déconseillé pour l'instant